js/templates/connectedPeersPage.html

```diff
@@ -1,17 +1,20 @@
-<div class="pad tx4">
-  <h1><%= ob.polyT('connectedPeersPage.heading') %></h1>
-  <p><em><%= ob.polyT('connectedPeersPage.infoMessage') %></em></p>
-
-  <div class="pageContent">
-    <div class="userPageFollow flexRow js-peerWrapper"></div>
-
-    <div class="js-morePeers hide">
-      <hr class="clrBr">
-      <a class="btn floR clrBr clrP js-morePeersBtn">Load More</a>
+<div class="pageContent">
+  <div class="rowLg">
+    <div class="flex">
+      <h1 class="flexExpand"><%= ob.polyT('connectedPeersPage.heading') %></h1>
+      <div class="tx3 btn"><%= ob.polyT('connectedPeersPage.totalPeers', { totalPeers: ob.peers.length }) %></div>
```

Comment:
I don't think this should get a btn class, since that is giving it a hover state.